### PR TITLE
Create update-on-pricing.md

### PR DESCRIPTION
**Pull request by haleysolomon**
-Added new FAQ page for update on pricing (ending grandfathering policy for free accounts)

### CODE REVIEW

File: articles/faq/update-on-pricing.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+The **Starter** plans are low-cost plans that make it easy for new or low-volume senders to get started. They include many of SparkPost’s great features, but some features are limited in order to keep costs low. See the chart below for a list of current feature limitations.
+The **Premier** plans are robust email packages for high-volume senders that include additional features to facilitate and manage sending large volumes of email. Feature breakdown of these plans can be found [here](https://www.sparkpost.com/docs/faq/difference-between-starter-and-premier/). 
+
+## If I cancel my plan and reactivate, can I go back to the old pricing? 
+
+No, if you would like to create a new account you will need to select from the currently available [plans and pricing](https://www.sparkpost.com/pricing/).
+
+## If I upgrade can I go back to my old plan?
+
+No, old plans will no longer be active/valid. You have the option to select any of the currently available paid plans or the current free “Test” plan with a monthly limit of 500 emails.
```
Comment (jgzamora) [inline articles/faq/update-on-pricing.md]: active/valid -> "available"

File: articles/faq/update-on-pricing.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+## Why did we change our pricing?
+
+Over the last four years, we have delivered significant expansions to our product such as [A/B testing](https://www.sparkpost.com/docs/tech-resources/a-b-testing-sparkpost/), [Recipient Validation](https://www.sparkpost.com/recipient-validation/), and our predictive email analytics platform [Signals](https://www.sparkpost.com/platform/signals/). And we’re shipping new features faster than ever. We believe that the new pricing is fair and in line with what is being offered in the market.
+
+## What are the differences between Starter and Premier plans?
+
+The **Starter** plans are low-cost plans that make it easy for new or low-volume senders to get started. They include many of SparkPost’s great features, but some features are limited in order to keep costs low. See the chart below for a list of current feature limitations.
```
Comment (jgzamora) [inline articles/faq/update-on-pricing.md]: There is no chart right?

File: articles/faq/update-on-pricing.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+The **Premier** plans are robust email packages for high-volume senders that include additional features to facilitate and manage sending large volumes of email. Feature breakdown of these plans can be found [here](https://www.sparkpost.com/docs/faq/difference-between-starter-and-premier/). 
+
+## If I cancel my plan and reactivate, can I go back to the old pricing?
```
Comment (jgzamora) [inline articles/faq/update-on-pricing.md]: Feel like we don't need this section or the section below can be If "I upgrade or cancel".

File: articles/faq/update-on-pricing.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
+---
+title: "Update on Pricing"
+description: "Information about pricing changes."
+---
+
+On **May 14, 2019**, we adjusted pricing for new self-serve plans.
+
+On **July 15, 2019**, we notified SparkPost customers on the old SparkPost “Free” plan of changes to our [plans and pricing](https://www.sparkpost.com/pricing/). On or before **September 15, 2019**, any customer on an old “Free” plan will need to select from one of our currently available plans to continue using SparkPost. _**Note:** You have the option to continue using SparkPost for free on a “Test” plan, but please take notice of the monthly volume limit of 500 emails._
```
Comment (jgzamora) [inline articles/faq/update-on-pricing.md]: I think we don't need the first sentence.

Comment (jgzamora) [reply articles/faq/update-on-pricing.md]: _**Note:** You have the option to continue using SparkPost for free on a “Test” plan, but please take notice of the monthly volume limit of 500 emails._
---- >
You can continue using SparkPost for free on our "Test" plan, which has a monthly limit of 500 emails.

File: articles/faq/update-on-pricing.md (hunk 3 above)
Comment (haleysolomon) [reply articles/faq/update-on-pricing.md]: Per convo with Ang based on feedback from Mitch, leaving both.

File: articles/faq/update-on-pricing.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
+---
+title: "Update on Pricing"
+description: "Information about pricing changes."
+---
+
+On **May 14, 2019**, we adjusted pricing for new self-serve plans.
```
[the remark this answers is not in the document]
Comment (haleysolomon) [reply articles/faq/update-on-pricing.md]: removed

File: articles/faq/update-on-pricing.md (hunk 1 above)
Comment (haleysolomon) [reply articles/faq/update-on-pricing.md]: done

File: articles/faq/update-on-pricing.md (hunk 2 above)
Comment (haleysolomon) [reply articles/faq/update-on-pricing.md]: Yeah thought I already deleted that? Don't see that line now.

**Pull request merged**